cli/uninstall/uninstall.go

```diff
@@ -112,12 +112,30 @@ func UninstallProgram(program string, programVersion string, binDst string, head
 			return err
 		}
 	}
+	// We need to try to delete all versions with 'v' prefix before x.y.z version
+	// and without it.
+	versionsToDelete := make([]string, 0, 2)
+	versionsToDelete = append(versionsToDelete, programVersion)
+
+	if program == search.ProgramTt &&
+		strings.Contains(programVersion, ".") && programVersion[0] != 'v' {
+		versionsToDelete = append(versionsToDelete, "v"+programVersion)
+	} else if program == search.ProgramTt &&
+		strings.Contains(programVersion, ".") && programVersion[0] == 'v' {
+		versionsToDelete = append(versionsToDelete, programVersion[1:])
+	}
 
 	var isSymlinkRemoved bool
-	isSymlinkRemoved, err = remove(program, programVersion, binDst, cmdCtx)
+	for _, verToDel := range versionsToDelete {
+		isSymlinkRemoved, err = remove(program, verToDel, binDst, cmdCtx)
+		if err == nil {
```

Comment:
If the first `remove` try failed, the loop continues for the second version string. So, we lost the info about the error occurred.

Reply:
Added handling an error excluding case when file was not found.